lib/LedgerSMB/Admin/Command/user.pm

```diff
@@ -445,9 +451,9 @@ __END__
 =head1 SYNOPSIS
 
    ledgersmb-admin user list <db-uri>
-   ledgersmb-admin user create <db-uri> [options] 
-   ledgersmb-admin user delete <db-uri>
-   ledgersmb-admin user change <db-uri>
+   ledgersmb-admin user create <db-uri> <user> [options] 
```

Comment:
Where is the `<user>` parsed from the command line arguments (and presumably stored in a variable)?